### ISSUE TITLE
pygtk and pygobject not creating .pth file

### ISSUE THREAD

**Issue by lee218llnl** [LLNL/spack#3045]
@tgamblin I'm not sure when this creeped back up, but I notice that pygtk and pygobject are not creating the appropriate gtk.pth and gobject.pth files in the site-packages directory:

[lee218@opal186:spack]$ ll /nfs/tmp2/lee218/pkg/spack/opt/spack/linux-rhel7-x86_64/gcc-4.9.3/py-pygtk-2.24.0-vwz4gz3lofwe436bjfzjdi3tgx7mcy63/lib/python2.7/site-packages/
total 32
drwx------ 3 lee218 lee218 2048 Feb  6 13:40 gtk-2.0

I thought having this path in $PYTHONPATH at build time, which Spack appears to be doing properly, used to suffice, so I'm not sure why this is broken. Anyone have any quick insights before I try to dig into this deeper?

**Comment by lee218llnl** [comment]
It appears that this is an issue with the latest gobject-3.0. The py-pygobject package will need to be updated b/c the pypi url for version 2.28 is no longer valid. I tried updating this to the latest 3.X version and that one fails to put in the .pth files. I found that if I revert to 2.28, the .pth files are generated correctly. I have a few other issues that I need to work out, but I will be putting in a PR for all of that soon. I will close this once I have that all ironed out.
